sources/HelpTab.cpp

```diff
@@ -137,10 +141,10 @@ HelpTab::HelpTab()
 	fInfo->MakeEditable(false);
 	fInfo->SetWordWrap(true);
 	fInfo->SetStylable(true);
-	fInfo->SetText("Filer is an automatic file organizer. It takes the "
+	fInfo->SetText(B_TRANSLATE("Filer is an automatic file organizer. It takes the "
 		"files it's opened with or that are dropped on it and moves, "
 		"renames, copies or does all sorts of other things with them "
-		"according to rules created by the user.");
+		"according to rules created by the user."));
 
```

Comment:
Maybe the text could be converted to a constant that can be re-used in the DropZone about window.
The copyrights should be translated too.